build.py

```diff
@@ -1,75 +1,180 @@
 # A nasty basic implementation with no tests to preserve the ability to build data required by the
 # service.
 
-import glob
+import argparse
```

Comment:
Why not using `click`? I find it more readable when writing the CLI. In addition, I hope we can use the built in runner for the tests, so you do not need to invoke `subprocess`.